gulpfile.js

```diff
@@ -46,10 +52,50 @@ gulp.task('clean', function () {
     .pipe(clean());
 });
 
+function bundle(dev) {
+  var modules = helpers.getArgModules(),
+      allModules = helpers.getModuleNames(modules);
+
+  if(modules.length === 0) {
+    modules = allModules;
+  } else {
+    var diff = _.difference(modules, allModules);
+    if(diff.length !== 0) {
+      throw new gutil.PluginError({
+        plugin: 'bundle',
+        message: 'invalid modules: ' + diff.join(', ')
+      });
+    }
+  }
+
+  var entries = [helpers.getBuiltPrebid(dev)].concat(helpers.getBuiltModules(dev, modules));
+
+  gutil.log('Concatenating files:\n', entries);
+  gutil.log('Appending ' + prebid.globalVarName + '.processQueue();');
+
+  return gulp.src(
+      entries
+    ).pipe(concat(argv.bundleName ? argv.bundleName : 'bundle.js'))
```

Comment:
FWIW, I like @mkendall07 's suggestion -- prebid-core.js instead of prebid.js and prebid.js instead of bundle.js